spec/controllers/v1/analyses_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'spec_helper'
+
+describe V1::AnalysesController do
+  render_views
+
+  let(:inventory) { FactoryGirl.create(:inventory) }
+
+  before :each do
+    request.env['HTTP_ACCEPT'] = 'application/json'
```

Comment:
Absolutely, and I think it would be the better approach. Again, following prevalent patterns. Let me know how you'd like me to proceed.